README.md

```diff
@@ -11,16 +13,57 @@ about writing tests.
 
 ## Running tests
 
-Install dependencies and run Locust:
+Install dependencies:
 
 ```shell
 poetry install
-poetry run locust
+```
+
+Write a [configuration file](#configuration-file), place its fully-qualified path in
+the `CIRCULATION_LOAD_CONFIGURATION_FILE` environment variable, and run Locust:
+
+```shell
+poetry run \
+  env CIRCULATION_LOAD_CONFIGURATION_FILE=$(realpath hosts.json) \
+  locust
 ```
 
 The locust dashboard will then be available at: `http://localhost:8089`.
 
-## Test details
+## Fair Warning
 
-The currently available load tests are documented inline in the
-[locustfile](locustfile.py).
+*Some tests involve borrowing books. Borrowing a book, on many CMs, will consume a license. This may cost money!*
+
+A good example is the [bookmark tests](src/circulation_load_test/common/cmsearchbookmark.py); in order to create
+bookmarks, the bookmarks must be associated with a loan. The bookmark test loans a random book, adds a lot of
+bookmarks, and then returns the book.
```

Comment:
It should be clear from the warning already here, but just to reiterate, maybe add a sentence warning that this should not be run against real production instances.